R/rstan_config.R

```diff
@@ -42,6 +42,11 @@ rstan_config <- function(pkgdir = ".") {
   # get stan model files
   stan_files <- list.files(file.path(pkgdir, "inst", "stan"),
                            full.names = TRUE, pattern = "\\.stan$")
+  stan_files <- c(stan_files,
+                  list.files(file.path(pkgdir, "inst", "stan"),
+                            full.names = TRUE,
+                            pattern = "\\.stanfunctions$",
+                            recursive = T))
```

Comment:
```suggestion
                            recursive = TRUE))
```

Reply:
Ah good catch! I was supposed to remove the `recursive` flag, since the behaviour should match that for `.stan` files proper (only looking in `inst/stan`, not any subdirectories)

Reply:
I've ended up just adding it to the pattern for the first `list.files`, rather than adding a second call